### PR TITLE
Show booking

### PR DESCRIPTION
**Pull request by nguyenthaingoc2002**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/66120)

## WHAT (optional)
- Show bookings.

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by nguyenthaingoc2002** [comment]
ready

File: app/views/bookings/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<% provide(:title, t("booking.all_bookings")) %>
```
Comment (anlvq-1058) [inline app/views/bookings/index.html.erb:1]: bỏ bớt () đi em

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by nguyenthaingoc2002** [comment]
ready

**Pull request merged**